src/index.css

```diff
@@ -1,27 +1,32 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;200;300;400;500;600;700&display=swap');
+
+/* CSS variables */
+:root {
+  --azure: #0290ff;
+  --opaque-azure: #4386bf;
+  --dark-white: #e8e8e8;
+  --white-color: #fff;
+  --black-color: #121212;
+  --warm-grey: #888;
+  --very-light-pink: #c4c4c4;
+  --main-font: 'Montserrat', sans-serif;
+  --second-font: 'Roboto Slab', serif;
+}
+
 body {
+  box-sizing: border-box;
+  font-size: 100%;
+  scroll-behavior: smooth;
   margin: 0;
-  font-family:
-    -apple-system,
-    BlinkMacSystemFont,
-    'Segoe UI',
-    'Roboto',
-    'Oxygen',
-    'Ubuntu',
-    'Cantarell',
-    'Fira Sans',
-    'Droid Sans',
-    'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  padding: 0;
+  text-decoration: none;
 }
 
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
 }
+
+/** { border: 1px solid black; } */
```

Comment:
- [ ] Good Job to implement some changes requested by the reviewer :+1:   But you need to work a bit on your design :cry: Please as the previous reviewer mentioned your design needs to be EXACTLY the same as Zeplin Bookstore application. As a developer in your future position It will be important to peer attention to details cause your future employer will judge you on it and following the design will make your app work fine and wonderful:smile:  

> * [ ]  The Chapter section should be aligned more to the right, kindly tweak your design so it looks exactly like the zeplin one
> 

#### Your design
![Screenshot from 2022-08-27 13-01-36](https://user-images.githubusercontent.com/6615394/187031379-743ab30e-3760-4e3a-8795-c31ac1998768.png)

#### Zeplin Bookstore design 
![Screenshot from 2022-08-27 12-28-29](https://user-images.githubusercontent.com/6615394/187031283-00f18795-1008-4662-b182-d21580f03756.png)
![Screenshot from 2022-08-27 12-28-41](https://user-images.githubusercontent.com/6615394/187031282-05d6fb9f-d87c-4a5f-9fc1-a34a1d041df8.png)

Reply:
Hello @codecaiine, thanks for your feedback, but I think I don't understand very well because I've already aligned the chapter section more at the right in the book component. Or maybe you want something like that?
![image](https://user-images.githubusercontent.com/53547956/187031868-d37d8ed3-6a98-480c-ad66-48543e94fb08.png)
Is it? Thanks for your answer.

Reply:
Hi @VaneCode . The previous reviewer requested you to align chapter section to the left more to the right and tweak your design so it looks exactly like the zeplin one . Because we can see some differences NOT only the chapter alignment :+1:

Reply:
@VaneCode Please add the remove your buttons **"Comments" "Remove" "Edit" .**  Also work on your navbar color design. Add the grey background color behind your Book card. :+1:

Reply:
Hello @codecaiine, thanks for your answer. I will implement your suggestions.

Reply:
Good to hear and good job :smile: